site/app/models/gradeable/Submitter.php

```diff
@@ -109,6 +109,14 @@ public function getRegistrationSection() {
         return $this->team_or_user->getRegistrationSection();
     }
 
+    /**
+     * Get the course_section_id for the submitter
+     * @return string
+     */
+    public function getCourseSectionId() {
+        return $this->team_or_user->getCourseSectionId();
+    }
+
```

Comment:
Again, please check on whether the function needs to be explicitly defined.

Reply:
This seems like it could be done without being explicitly defined, but it looks like none of the functions in submitter.php are this way. I think for consistency it's maybe better to leave it here.